app/models.py

```diff
@@ -46,6 +58,13 @@ class Place(db.Model):
     reviews:   so.Mapped[List[Review]] = so.relationship(
         back_populates='place')
     name:      so.Mapped[str] = so.mapped_column(sa.String(64))
+    rating:    so.Mapped[float] = so.mapped_column(sa.Float(asdecimal=True, precision=2), default=0)
+    weighted_rating: so.Mapped[float] = so.mapped_column(sa.Float(asdecimal=True, precision=2), default=0)
+    user_rating_count: so.Mapped[int] = so.mapped_column(sa.Integer, default=0)
+    summary:    so.Mapped[str] = so.mapped_column(sa.String(4096), default='')
+    updated_at: so.Mapped[datetime] = so.mapped_column(TZDateTime)
```

Comment:
Maybe we can put a default here too?
```py
from sqlalchemy import func
```
and then
```suggestion
    updated_at: so.Mapped[datetime] = so.mapped_column(TZDateTime, default=func.now(), onupdate=func.now())
```
Then we don't have to put updated_at anywhere